argschema/fields/files.py

```diff
@@ -7,8 +7,10 @@
 
 def validate_outpath(path):
     try:
-        with tempfile.TemporaryFile(mode='w', dir=path) as tfile:
+        with tempfile.NamedTemporaryFile(mode='w', dir=path) as tfile:
             tfile.write('0')
+            tfile.close()
```

Comment:
Does tfile.close() really make a difference?  You are already using the context manager in the proper way; it should close the file implicitly.